src/screens/Auth/screens/Login/components/LoginForm.tsx

```diff
@@ -34,9 +33,15 @@ const LoginForm: React.FunctionComponent<LoginFormProps> = ({ onSuccess }) => {
     });
 
     const onSubmit = async (data: LoginFormData) => {
-        const res = await withErrors<LoginFormData>(dispatch(login(data)).unwrap(), setError);
-        if (res !== false) {
-            onSuccess && onSuccess();
+        try {
+            const res = await withErrors<LoginFormData>(login(data).unwrap(), setError);
```

Comment:
The withErrors function now returns Promise.reject(error) instead of false, but the code still checks for false. This will always call onSuccess() after a catch block runs